### PR TITLE
add script to generate v3 ea

### PR DESCRIPTION
**Pull request by karen-stepanyan**
## Description

EA framework now provides interactive tool to generate new v3 adapters. This PR adds  a functionality to use that tool in monorepo. 

## Changes

- two new devDeps in main package.json, `@chainlink/external-adapter-framework` and `yo`
- `new` script is changed to use ea-framework for adapter generation
- packages/examples is removed
- Docs are modified 

## Steps to Test
To create test adapter, run 

`yarn new` 

optionally you can specify also the type (source is the default)

`yarn new composite`

### CODE REVIEW

**Review by alejoberardino — CHANGES_REQUESTED** [review]
Only change would be to directly remove the previous generation script and keep this as the only one. If preferred, this PR could just change the reference and we could open a separate PR to remove the old v2 generation

**Comment by karen-stepanyan** [comment]
hey @alejoberardino . Yeah, i think it  makes sense to replace it here so that we don't have a new script. 
I also deleted examples folder and updated the contributing file

**Review by alejoberardino — APPROVED** [review]
Approved, left one comment that can be ignored if it's not relevant

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,37 +17,28 @@ Thank you for your interest in improving the Chainlink External Adapter codebase
...
 For example
 
 ```bash
-$ yarn new source my-new-adapter
+$ yarn new source
 ```
 
 _If on a Mac, this requires `gnu-sed` to be installed and set as the default for the command `sed`._
 
 You can open a PR with the [New EA PR Template](./.github/PULL_REQUEST_TEMPLATE/new_ea_pr_template.md) by replacing `<branch>` in this URL: [https://github.com/smartcontractkit/external-adapters-js/compare/develop...<branch>?quick_pull=1&template=new_ea_pr_template.md](https://github.com/smartcontractkit/external-adapters-js/compare/develop...<branch>?quick_pull=1&template=new_ea_pr_template.md)
```
[the remark this answers is not in the document]
Comment (jacksnipe24) [reply CONTRIBUTING.md:40]: I think not only CONTRIBUTING.md but also other doc files need to be reviewed and updated. At least the PR opening checklist is not up to date at this point and also `gnu-sed` seems is not installed on my machine and it's not a problem at all.  Not sure if this is a right PR for them tho.

[the remark this answers is not in the document]
Comment (karen-stepanyan) [reply CONTRIBUTING.md:40]: I think not only CONTRIBUTING.md but also other doc files need to be reviewed and updated. At least the PR opening checklist is not up to date at this point and also gnu-sed seems is not installed on my machine and it's not a problem at all. Not sure if this is a right PR for them tho.

**Pull request merged**